defaults/main.yml

```diff
@@ -69,13 +69,6 @@ gitlab_enable_pages: '{{ gitlab_version | version_compare("8.17",
 # Application FQDN and DNS addresses [[[
 # --------------------------------------
 
-# .. envvar:: gitlab__fqdn [[[
-#
-# The Fully Qualified Domain Name of the GitLab application. This address is
-# used to configure the webserver frontend.
-gitlab__fqdn: 'code.{{ gitlab_domain }}'
-
-                                                                   # ]]]
```

Comment:
I think that this is my fault for not updating the `nginx` configuration properly. The idea for these variables is like this:

- `gitlab__domain` contains the equivalent of `ansible_domain` exposed via default variable so thait it can be overriden if desired; currently it's a list which is a mistake and should be changed to a string.

- `gitlab__fqdn` is the Fully Qualified Domain Name configured in `nginx` that uses `gitlab__domain` as the domain part.

Could you add back this variable? Then I can merge this PR and update the `nginx` configuration properly.